composeApp/src/commonMain/kotlin/com/jetbrains/kmpapp/screens/detail/DetailScreen.kt

```diff
@@ -34,12 +34,13 @@ import cafe.adriel.voyager.core.screen.Screen
 import cafe.adriel.voyager.koin.getScreenModel
 import cafe.adriel.voyager.navigator.LocalNavigator
 import cafe.adriel.voyager.navigator.currentOrThrow
-import com.jetbrains.kmpapp.MR
 import com.jetbrains.kmpapp.data.MuseumObject
 import com.jetbrains.kmpapp.screens.EmptyScreenContent
-import dev.icerock.moko.resources.compose.stringResource
 import io.kamel.image.KamelImage
 import io.kamel.image.asyncPainterResource
+import kmp_app_template.composeapp.generated.resources.Res
```

Comment:
@zsmb13 
According to [this](https://github.com/JetBrains/compose-multiplatform/blob/master/CHANGELOG.md#breaking-changes), you also need to add:
```kotlin
import kmp_app_template.composeapp.generated.resources.*
```

Reply:
That's correct, left some out-of-date imports there. Fixed now.